langgraph_supervisor/supervisor.py

```diff
@@ -54,6 +57,18 @@ def _make_call_agent(
     add_handoff_back_messages: bool,
     supervisor_name: str,
 ) -> Callable[[dict], dict] | RunnableCallable:
+    """Wrap the supervisor agent to modify the state to be passed to the worker/sub-agent.
```

Comment:
this docstring is incorrect - this is wrapping individual agents, not the supervisor agent, and modify the state that is passed back to the supervisor

the implementation below should work either way, because `output` in `process_output` will contain the full history, since we effectively do `subgraph.invoke({... full_history}) -> full_history + agent output`

i would double check that it works properly with parallel handoffs

Reply:
ha ya good callout - bad decision for me to vibecode a docstring